extract_test.go

```diff
@@ -1,14 +1,14 @@
 package extract_test
 
 import (
-	"bytes"
+	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
-	"github.com/codeclysm/extract"
+	"github.com/AlessandroSanino1994/extract"
```

Comment:
I can't merge it like this. It would break everything.

You should work in the folder github.com/codeclysm/extract by adding your own remote